<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>611</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1" rowspan="2">
     <widget class="CanvasWidget" name="Canvas" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>4</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>781</width>
        <height>536</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QTreeWidget" name="treeWidget">
      <property name="minimumSize">
       <size>
        <width>195</width>
        <height>500</height>
       </size>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::CustomContextMenu</enum>
      </property>
      <column>
       <property name="text">
        <string notr="true">1</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="FileMenu">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="NewAction"/>
    <addaction name="OpenAction"/>
    <addaction name="SaveAction"/>
    <addaction name="SaveAsAction"/>
    <addaction name="separator"/>
    <addaction name="ExitAction"/>
   </widget>
   <widget class="QMenu" name="ListMenu">
    <property name="title">
     <string>&amp;List</string>
    </property>
    <widget class="QMenu" name="ListResultMenu">
     <property name="title">
      <string>&amp;Result</string>
     </property>
     <addaction name="ListNodeResultAction"/>
     <addaction name="ListElementResultAction"/>
     <addaction name="ListConstraintResultAction"/>
    </widget>
    <addaction name="MaterialsListAction"/>
    <addaction name="SectionsListAction"/>
    <addaction name="OffsetsListAction"/>
    <addaction name="NodesListAction"/>
    <addaction name="ElementsListAction"/>
    <addaction name="ConstraintsListAction"/>
    <addaction name="LoadsListAction"/>
    <addaction name="separator"/>
    <addaction name="ListResultMenu"/>
   </widget>
   <widget class="QMenu" name="HelpMenu">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="UserGuideAction"/>
    <addaction name="AboutAction"/>
    <addaction name="AboutQtAction"/>
   </widget>
   <widget class="QMenu" name="PlotMenu">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>&amp;Plot</string>
    </property>
    <widget class="QMenu" name="PlotResultMenu">
     <property name="title">
      <string>&amp;Results</string>
     </property>
     <addaction name="DeformedShapeAction"/>
     <addaction name="ElementInformationAction"/>
     <addaction name="ConstraintForcesAction"/>
    </widget>
    <widget class="QMenu" name="menuUser_Defined_Shapes">
     <property name="title">
      <string>User &amp;Defined Shapes</string>
     </property>
     <addaction name="PlotDefinedAction"/>
     <addaction name="ShowNodesAction"/>
     <addaction name="ShowElementsAction"/>
     <addaction name="ShowConstraintsAction"/>
     <addaction name="ShowLoadsAction"/>
     <addaction name="ShowResetAction"/>
    </widget>
    <addaction name="FitAction"/>
    <addaction name="ShowLabelsAction"/>
    <addaction name="ShowAxisAction"/>
    <addaction name="separator"/>
    <addaction name="menuUser_Defined_Shapes"/>
    <addaction name="separator"/>
    <addaction name="PlotResultMenu"/>
   </widget>
   <addaction name="FileMenu"/>
   <addaction name="ListMenu"/>
   <addaction name="PlotMenu"/>
   <addaction name="HelpMenu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="NewAction">
   <property name="text">
    <string>&amp;New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="OpenAction">
   <property name="text">
    <string>&amp;Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="SaveAction">
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="ExitAction">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="FitAction">
   <property name="text">
    <string>&amp;Fit View</string>
   </property>
  </action>
  <action name="ShowAxisAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show &amp;Axis</string>
   </property>
  </action>
  <action name="ShowLabelsAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show &amp;Labels</string>
   </property>
  </action>
  <action name="SaveAsAction">
   <property name="text">
    <string>Save &amp;As</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="AboutAction">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="AboutQtAction">
   <property name="text">
    <string>About &amp;Qt</string>
   </property>
  </action>
  <action name="actionShow_Nodes">
   <property name="text">
    <string>Show Nodes</string>
   </property>
  </action>
  <action name="actionShow_Rods">
   <property name="text">
    <string>Show Rods</string>
   </property>
  </action>
  <action name="ShowNodesAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Plot &amp;Nodes</string>
   </property>
  </action>
  <action name="ShowElementsAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Plot &amp;Elements</string>
   </property>
  </action>
  <action name="ShowConstraintsAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Plot &amp;Constraints</string>
   </property>
  </action>
  <action name="ShowLoadsAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Plot &amp;Loads</string>
   </property>
  </action>
  <action name="ShowResetAction">
   <property name="text">
    <string>&amp;Reset</string>
   </property>
  </action>
  <action name="ElementInformationAction">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Element Information</string>
   </property>
  </action>
  <action name="DeformedShapeAction">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Deformed Shape</string>
   </property>
  </action>
  <action name="ConstraintForcesAction">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Constraint Forces</string>
   </property>
  </action>
  <action name="ShowConsoleAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Show &amp;Console</string>
   </property>
  </action>
  <action name="PlotDefinedAction">
   <property name="text">
    <string>&amp;Plot</string>
   </property>
  </action>
  <action name="NodesListAction">
   <property name="text">
    <string>&amp;Nodes</string>
   </property>
  </action>
  <action name="ElementsListAction">
   <property name="text">
    <string>&amp;Elements</string>
   </property>
  </action>
  <action name="ConstraintsListAction">
   <property name="text">
    <string>&amp;Constraints</string>
   </property>
  </action>
  <action name="LoadsListAction">
   <property name="text">
    <string>&amp;Loads</string>
   </property>
  </action>
  <action name="ListNodeResultAction">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Nodes Displacement</string>
   </property>
  </action>
  <action name="ListElementResultAction">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Elements Information</string>
   </property>
  </action>
  <action name="ListConstraintResultAction">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Constraint Forces</string>
   </property>
  </action>
  <action name="MaterialsListAction">
   <property name="text">
    <string>&amp;Materials</string>
   </property>
  </action>
  <action name="SectionsListAction">
   <property name="text">
    <string>&amp;Sections</string>
   </property>
  </action>
  <action name="OffsetsListAction">
   <property name="text">
    <string>&amp;Offsets</string>
   </property>
  </action>
  <action name="UserGuideAction">
   <property name="text">
    <string>User &amp;Guide</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CanvasWidget</class>
   <extends>QWidget</extends>
   <header>canvaswidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>ExitAction</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>499</x>
     <y>399</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
